target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -106,6 +106,9 @@ case "$FIRMWARE" in
 	unifiac-pro)
 		ath10kcal_extract "EEPROM" 20480 2116
 		;;
+	rb-952ui-5ac2nd)
+		ath10kcal_from_file "/sys/firmware/routerboot/ext_wlan_data" 20480 2116
+		;;
```

Comment:
The MAC address is set correctly for both radios. I assume that in the ath10k case this is contained in the EEPROM along with caldata.